persistence/src/main/resources/db/migration/V1.14__Add_executed_test_tasks_array_to_build.sql

```diff
@@ -0,0 +1,2 @@
+alter table build
+    add executed_test_tasks varchar(255) not null default '{}'
```

Comment:
```suggestion
    add executed_test_tasks varchar(255) [] not null default '{}'
```

Isn't the `[]` missing here?

Reply:
Yeah... the `[]` was lost mistakenly when I added `not null` clause.